composer.json

```diff
@@ -22,11 +22,13 @@
             "email": "bordun.alexandr@gmail.com"
         }
     ],
-    "autoload": {
-        "psr-4": {"borales\\widgets\\ace\\": "src/"}
-    },
     "require": {
-        "yiisoft/yii2": "*",
-        "bower-asset/ace-builds": "~1.1"
+        "yiisoft/yii2": "~2.0.5",
```

Comment:
@wbraganca `~2.0.5` is it really necessary? Is it not compatible with earlier versions?